File: Front/client/src/DTOs/CommentDTO.ts
import { v4 as uuidv4 } from 'uuid';
export class CommentDTO{
    id: number;
    guid: string;
    comment: string;
    userId: number;
    postId:number;
    constructor(){
        this.id = 0;
        this.comment = "";
        this.userId = 0;
        this.postId = 0;
        this.guid = "";
    }
    generatedGuid(){
        this.guid = uuidv4()
    }
}

File: BackEnd/Api/DTOs/CommentDTO.cs
namespace Api.DTOs
{
    public class CommentDTO
    {
        public int Id { get; set; }
        public string guid {get; set;}
        public string comment { get; set; }
        public int userId { get; set; }
        public int postId { get; set; }
        public DateTime dateCreate {get; set;}
    }
}
